include/cantera/zeroD/ReactorNet.h

```diff
@@ -207,6 +207,22 @@ class ReactorNet : public FuncEval
         m_integrator_init = false;
     }
 
+    /** Set the maximum number of internal integration time-steps the
+     *  integrator will take before reaching the next output time
+     *  @param nmax The maximum number of steps, setting this value
+     *              to zero disables this option.
+     */
+    virtual void setMaxSteps(int nmax) {
+        m_integ->setMaxSteps(nmax);
+    }
+
+    /** Returns the maximum number of internal integration time-steps the
+     *  integrator will take before reaching the next output time
+     */
```

Comment:
If there's only a 'brief' docstring, I tend to prefer just using the `//!` on each line rather than the C-style block comment.